File: Baekjoon/BaaarkingDog/0x02_basic/2587.cpp
// 2587: 대표값2

#include <iostream>

int main() {
  int nums[5], sum = 0;
  
  for (int i=0; i<5; i++) {
    std::cin >> nums[i];
    sum += nums[i];
  }

  for (int i=0; i<5-1; i++) {
    for (int j=0; j<5-1-i; j++) {
      if (nums[j] > nums[j+1]) {
        int temp = nums[j];
        nums[j] = nums[j+1];
        nums[j+1] = temp;
      }
    }
  }

  std::cout << sum / 5 << "\n" << nums[2];

  return 0;
}

File: Baekjoon/BaaarkingDog/0x02_basic/2587.py
# 2587: 대표값2

nums = [0 for _ in range(5)]
sum = 0

for i in range(5):
  nums[i] = int(input())
  sum += nums[i]

nums = sorted(nums)

print(f"{int(sum/5)}\n{nums[2]}")